content/projects/gsoc/2020/project-ideas/jenkins-x-openwrt-ppa.adoc

```diff
@@ -0,0 +1,60 @@
+---
+layout: gsocprojectidea
+title: "PPA for OpenWRT"
+goal: "Create a platform to build and host packages for OpenWRT (PPA, or Personal Package Archive) using Jenkins X"
+category: Jenkins X
+year: 2020
+status: published
+skills:
+- Shell
+- Makefiles
+- Docker
+- Kubernetes
+mentors:
+- "zoobab"
+links:
+  web: https://www.openwrt.org
```

Comment:
Do you have special channels defined for your project? With the current setup it will be GSoC Gitter and the public mailing list defined as contacted points (defaults)